client/src/components/DataFiles/DataFilesSystemSelector/DataFilesSystemSelector.js

```diff
@@ -34,7 +34,7 @@ const DataFilesSystemSelector = ({ systemId, section, disabled }) => {
     <>
       <DropdownSelector
         onChange={openSystem}
-        value={selectedSystem}
+        value={selectedSystem || ''}
```

Comment:
Would it make more sense to apply the `''` to the [default prop for `systemId`](https://github.com/TACC/Frontera-Portal/pull/262/files#diff-888a79ba1256c305b0920984872e7e881937fa9dfc03ec398d2faf90476d2719R58) if the purpose is to avoid the error for the test?

Reply:
I agree with Sal. If we revert the changes to this line and make the changes [line 58](https://github.com/TACC/Frontera-Portal/pull/262/files#diff-888a79ba1256c305b0920984872e7e881937fa9dfc03ec398d2faf90476d2719L58) instead (`systemId: '',`) this will also pass the tests.

Reply:
Thanks, y'all! Updated.